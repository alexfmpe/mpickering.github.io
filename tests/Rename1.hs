

foo x y =
   do c <- getChar
      return c

