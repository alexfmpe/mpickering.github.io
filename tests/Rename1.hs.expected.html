

bar2 x y =
   do c <- getChar
      return c

